test/explorer/swagger-explorer.spec.ts

```diff
@@ -829,14 +843,15 @@ describe('SwaggerExplorer', () => {
       expect(routes[0].root.parameters).toEqual([
         {
           in: 'query',
+          isArray: true,
```

Comment:
This property shouldn't be defined here^

Reply:
In this test it should shouldn't it? 

Cause in the first Foo Controller it is defined in the ApiQuery decorator. 
```@ApiQuery({ name: 'page', enum: ['d', 'e', 'f'], isArray: true })```

The test following it it is excluded which is how it is supposed to work no?
```@ApiQuery({ name: 'page', enum: ['d', 'e', 'f'] })```